### PR TITLE
Add recruitment forms, update wording of positions

### PR DESCRIPTION
**Pull request by bobheadxi**
The forms are here:

https://drive.google.com/drive/u/2/folders/1OCGLGKXjq48NBSmYFOGRLLSlpCxpOCe8

preview:

https://deploy-preview-69--ubclaunchpad.netlify.com/

related to https://github.com/ubclaunchpad/tech-leads/issues/10

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -421,46 +421,55 @@ <h1>Join Us</h1>
       please don't hesitate to get in contact with us by emailing
       <a href="mailto:presidents@ubclaunchpad.com">presidents@ubclaunchpad.com</a>.
     </p>
-    <!-- <br />
+    <br />
     <p class="large">
-      Applications are currently open! Apply by 12AM on Saturday, September 8th.
+      Applications are currently open!
```
Comment (frostyshadows) [inline index.html]: We should still have a clear deadline, maybe first Saturday back in January?

File: index.html [hunk 2]
```diff
@@ -421,46 +421,55 @@ <h1>Join Us</h1>
...
+
       <div class="col new-position">
         <div class="info">
-          <h2>Developer</h2>
+          <h2>Designer</h2>
           <br />
           <p class="small">
-            Join our development team to improve your skills and resume. You have the basics down, but want to collaborate with a team
-            to build awesome software and learn professional development practices.
+            Realize your design ideas with a dedicated development team, and learn
```
Comment (frostyshadows) [inline index.html]: I know you didn't actually change any of the copy but maybe with a "dedicated project team" instead

File: index.html [hunk 3]
```diff
@@ -421,46 +421,55 @@ <h1>Join Us</h1>
...
       <div class="col new-position">
         <div class="info">
-          <h2>Business</h2>
+          <h2>Strategy</h2>
           <br />
           <p class="small">
-            Join our business strategy team. Develop industry relationships and insight while securing funding and sponsorships.
+            Join our business strategy team. Develop industry relationships and
+            insight while securing funding and sponsorships. Work closely with
+            partners to organize events.
```
Comment (frostyshadows) [inline index.html]: adding a bit more detail to what the events are would be nice. something like "Work closely with partners to organize and market events contributing to UBC's tech community."

**Review by frostyshadows — DISMISSED** [review]

File: index.html (hunk 3 above)
Comment (bobheadxi) [reply index.html]: 5309d40

File: index.html (hunk 2 above)
Comment (bobheadxi) [reply index.html]: 775e859

File: index.html (hunk 1 above)
Comment (bobheadxi) [reply index.html]: cb9b45a

File: index.html [hunk 4]
```diff
@@ -416,51 +416,68 @@ <h1 class="team">Our Team</h2>
...
           <br />
           <p class="small">
-            Join our development team to improve your skills and resume. You have the basics down, but want to collaborate with a team
-            to build awesome software and learn professional development practices.
+            Realize your design ideas with a dedicated project team, and learn
+            from peers in a talented team of experienced designers. You're full
+            of ideas and you're ready to guide the development of a project from
+            conception to release.
+
+            One example of the design team's most recent projects is our new
```
Comment (frostyshadows) [inline index.html:443]: 🎉

**Review by frostyshadows — APPROVED** [review]

**Pull request merged**